lib/cortex/snippets/client/helper.rb

```diff
@@ -2,11 +2,13 @@ module Cortex
   module Snippets
     module Client
       module Helper
-        def snippet(options = {}, &block)
+        def snippet(client, options = {}, &block)
+          @cortex_client = client
+
           snippets = webpage[:snippets] || []
           snippet = snippets.find { |snippet| snippet[:document][:name] == options[:id] }
 
-          if snippet.nil? || snippet[:document][:body].nil? || snippet[:document][:body].empty?
+          if snippet.nil? || snippet[:document][:body].blank?
```

Comment:
The infrastructure is in place for this to support servers other than Rails (to potentially support teams other than ours), so there's no guarantee this ActiveSupport-provided helper will exist